nengo/tests/test_learning_rules.py

```diff
@@ -138,8 +173,26 @@ def test_pes_neuron_ens(Simulator, plt, seed, rng, allclose):
         allclose,
         pre_neurons=True,
         post_neurons=False,
+        weight_solver=weight_solver,
+        n=n,
+        transform=initial_weights,
+    )
+
+
+def test_pes_ens_neurons(Simulator, plt, seed, rng, allclose):
+    n = 200
+    initial_weights = np.ones((n, 2))
+    _test_pes(
+        Simulator,
+        nengo.LIF,
+        plt,
+        seed,
+        allclose,
+        pre_neurons=False,
+        post_neurons=True,
         n=n,
         transform=initial_weights,
+        rate=1e-4,
```

Comment:
It seems like we should be testing weight solvers here.

Reply:
These pes tests are kind of expensive, so I ended up adding a new test (see `test_build_pes`) that doesn't run any actual learning, just checks that these build with the correct shapes we expect (since that's where these different cases were failing).